griptape/tools/google_docs/tool.py

```diff
@@ -0,0 +1,270 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING
+import logging
+from attr import field, define
+from schema import Schema, Optional, Literal
+from griptape.artifacts import ErrorArtifact, InfoArtifact, BlobArtifact, ListArtifact
+from griptape.utils.decorators import activity
+from griptape.tools import BaseGoogleClient
+
+if TYPE_CHECKING:
+    from googleapiclient.discovery import Resource
+
+@define
+class GoogleDocsClient(BaseGoogleClient):
+    DOCS_SCOPES = ["https://www.googleapis.com/auth/documents"]
+
+    DRIVE_SCOPES = ["https://www.googleapis.com/auth/drive.file"]
+
+    owner_email: str = field(default=None)
+
+    @activity(
+        config={
+            "description": "Can be used to create a new Google Doc in Drive",
```

Comment:
Is there a reason to mention "Drive" here?

Reply:
Nope, removed it!